docs/docs/hive.md

```diff
@@ -145,17 +130,14 @@ The table level configuration overrides the global Hadoop configuration.
 
 ##### Hive on Tez configuration
 
-To use the Tez engine on Hive `3.1.2` or later, Tez needs to be upgraded to >= `0.10.1` which contains a necessary fix [TEZ-4248](https://issues.apache.org/jira/browse/TEZ-4248).
-
-To use the Tez engine on Hive `2.3.x`, you will need to manually build Tez from the `branch-0.9` branch due to a
-backwards incompatibility issue with Tez `0.10.1`.
-
-In both cases, you will also need to set the following property in the `tez-site.xml` configuration file: `tez.mrreader.config.update.properties=hive.io.file.readcolumn.names,hive.io.file.readcolumn.ids`.
+To use the Tez engine, you will need to set the following property in the `tez-site.xml` configuration file: `tez.mrreader.config.update.properties=hive.io.file.readcolumn.names,hive.io.file.readcolumn.ids`.
```

Comment:
I don't think we need this in Hive 4.
@deniskuzZ: Could you please confirm?

Reply:
MR is deprecated since Hive-4.0

Reply:
i don't think you need to set `tez.mrreader.config.update.properties` manually, those are set in `HiveIcebergStorageHandler` (HIVE-25581)